### DISCUSSION TITLE
Quasar 2 + Sentry

### DISCUSSION THREAD

**Discussion by Zeggaroth** [quasarframework/quasar#11932] [Ideas / Proposals]
With the new update with Quasar 2, the documentation from Sentry for Vue 3 doesn't seem to be compatible with how it was in Quasar 1 with the integration inside the Boot.
https://docs.sentry.io/platforms/javascript/guides/vue/

Is it possible to share any specific documentation available or good advice to implement Sentry inside a Quasar 2 project ?

**Comment by MilosPaunovic** [comment]
[https://docs.sentry.io/platforms/javascript/guides/vue/#vue-3](https://docs.sentry.io/platforms/javascript/guides/vue/#vue-3)
